setup/certificates.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+source ./environment.sh
+
+OPTIONS=":a:h:k:p:w:"
```

Comment:
Option "k"  doesn't seem to be defined below

Reply:
Removed, thanks.